samples/fluent.js

```diff
@@ -18,7 +18,7 @@
 const express = require('express');
 const app = express();
 
-app.get('*', function(req, res, next) {
+app.get('*', (req, res, next) => {
   return next('oops');
```

Comment:
Would it make sense to use short hand notation and leave out {} and return?

Reply:
Sure!  Will cover in a follow up.